common/views/ProgressBar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import pptColors from '../styles/color';
+import { getBarWidthPercent } from '../utils/';
+
+export default class ProgressBar extends React.Component {
+
+  static propTypes = {
+    stats: React.PropTypes.object,
+  }
+
+  getStyles() {
+    return {
+      root: {
+        height: 10
+      },
+      individual: {
+        display: 'inline-block',
```

Comment:
Using `inline-block` here causes the block's vertical position affected by `line-height`.
Using `flexbox` on their parent or using `float` on themselves is recommanded.